pages/index.js

```diff
@@ -14,16 +15,17 @@ export default function Home() {
   const [nameTF, setNameTF] = useState("");
   const [powerTF, setPowerTF] = useState("");
   const [healthTF, setHealthTF] = useState("");
+  const [sigilsTF, setSigilsTF] = useState("");
 
   // Stagger requests so they only send 500ms after user stops typing
   useEffect(() => {
     const timer = setTimeout(() => {
       setBusy(true);
-      const transformations = [nameTF, powerTF, healthTF].join();
+      const transformations = [nameTF, powerTF, healthTF, sigilsTF].join("");
       setUrl(`${CLOUDINARY_BASE}${transformations}${CARD_BASE}`);
     }, 500);
     return () => clearTimeout(timer);
-  }, [nameTF, powerTF, healthTF]);
+  }, [nameTF, powerTF, healthTF, sigilsTF]);
```

Comment:
Could the request staggering and timer stuff be abstracted out to a 'useDebounce' hook or something similar?

Reply:
It proooobably could 🤔 I had a look at decorator functions and the like because there was a point where I had this snippet of code repeated in each editor, but now it's standardised I should be able to move it someplace nicer. Will add a card to `spike` this, but is not relevant to this PR ^^

Reply:
(https://trello.com/c/4mu1x6fp/35-investigate-moving-timer-state-to-a-usetimer-hook-or-similar)

Reply:
Yeah, that's fair. Just figured it would make a good reusable custom hook for sure.